src/seedu/addressbook/AddressBook.java

```diff
@@ -1146,12 +1152,12 @@ private static String getUsageInfoForExitCommand() {
     /**
      * Removes sign(p/, d/, etc) from parameter string
```

Comment:
Header comments needs to be updated to reflect the modified behaviour. 

When you modify the behavior, remember to update documentation and tests as well.